client/src/app.js

```diff
@@ -1,14 +1,33 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import Card from './components/card/card.js'
+import React, { Component } from "react";
+import {
+  BrowserRouter as Router,
+  Link,
```

Comment:
Please delete `Link, NavLink, Redirect` and `Prompt` as you're not using them.

Reply:
we will need then in the future.